src/orderproducerservice/deployment-staging.yaml

```diff
@@ -50,6 +50,7 @@ spec:
                     operator: In
                     values:
                       - ng-4
+                      - ng-7
```

Comment:
This will not deploy to both node groups, but either ng-4 or ng-7. 

This was supposed to be determined dynamically based on the `NODE_GROUP` env [here](https://github.com/DataDog/opentelemetry-demo/blob/prod/ci/scripts/ci-deploy-demo-staging.sh#L39), but there is a bug in how it is currently. 

I've merged a PR here: #41 that fixes this. Can you can merge prod in this PR and then you can remove the changes to `src/orderproducerservice/deployment-staging.yaml` and
`src/zookeeperservice/deployment-staging.yaml` as it will be read from the env var `NODE_GROUP: ng-7` that you set.

Reply:
Great, I merged the upstream change